### PR TITLE
Minor typos fixed along with additional note

### PR DESCRIPTION
**Pull request by poltak**
- Typo where words were repeated (OCD)
- Note about python-dev requirement for streamparse build (I had issues without these installed, hence the note may help other newbies)

### CODE REVIEW

File: doc/source/quickstart.rst [hunk 1]
```diff
@@ -34,6 +34,8 @@ Once that's all set, you install streamparse using ``pip``::
 
     > pip install streamparse
 
+Note that installing streamparse depends on python-dev libs, hence make
+sure that your distro's python-dev package are also installed.
```
Comment (dan-blanchard) [inline doc/source/quickstart.rst]: We don't build any C extensions or anything, so I don't see why this would be true.

**Comment by poltak** [comment]
On the system I was testing with (Ubuntu 14.04.2 LTS), it required the python-dev package to be installed first, as it was failing on build with 
```
pip install streamparse
```
as it could not find the "Python.h" header.

Let me know if you need any further info.

**Comment by dan-blanchard** [comment]
I would need to see the whole output of `pip` before it failed, because it's almost definitely from one of our prereqs, and not us.

**Comment by poltak** [comment]
Yeah, you are right, Dan. After looking again, it is needed for both the markupsafe and pycrypto packages, which either streamparse directly or indirectly depends on.

**Pull request closed without merging**